library/init/meta/tactic.lean

```diff
@@ -896,8 +896,13 @@ meta def intron_with
 
 /-- Returns n fully qualified if it refers to a constant, or else fails. -/
```

Comment:
Does this actually fail anymore? I think it will just return garbage.

Reply:
That's true, it will return `anonymous`. We could make it crash too, to be conservative.

Reply:
At least the `alias` tactic in mathlib depends on `resolve_constant` failing (if the declaration cannot be found).

Reply:
I made the change. Is there a way to launch a build of mathlib with this version of Lean?

Reply:
The way I've always done it is to change the `leanpkg.toml` of mathlib to point to a local elan toolchain (try `elan toolchain link --help`).